invenio_rdm_records/services/components/pids.py

```diff
@@ -125,3 +125,41 @@ def edit(self, identity, draft=None, record=None):
         pids = record.get("pids", {})
         self.service.pids.pid_manager.validate(pids, record)
         draft.pids = pids
+
+
+class ParentPIDsComponent(ServiceComponent):
```

Comment:
This class is very similar to the one above, with the difference that we only care about creation and publishing when creating/updating the concept PIDs.